src/app/modals/login-pop-up/login-pop-up.component.html

```diff
@@ -0,0 +1,35 @@
+<form [formGroup]="loginForm" (ngSubmit)="onSubmit()">
+  <div class="login-pop-up-frame16">
+    <div class="relative">
+      <a class = "close" mat-button [mat-dialog-close]="true"></a>
+    </div>
+    <span class="login-pop-up-text">Sign in to Pomodoro</span>
+    <div class="login-pop-up-group9">
+      <span class="login-pop-up-text02">Not registered yet?</span>
+      <a class="login-pop-up-text04">Sign up</a>
+    </div>
+        <div class="login-pop-up-group6">
+          <label class = "login-pop-up-text06" for="email">Your email</label>
+          <input class = "login-pop-up-email" id="email" formControlName="email">
+        </div>
+        <div class="login-pop-up-group8">
+          <label class = "login-pop-up-text08" for = "password">Your password</label>
+          <input class = "login-pop-up-password" id = "password" formControlName="password">
```

Comment:
Just a small remark: I think it would be better to specify types for email and password inputs

Reply:
That's a good idea. I'll change it